mlflow/utils/rest_utils.py

```diff
@@ -45,6 +46,9 @@
 _V3_REST_API_PATH_PREFIX = "/api/3.0"
 _V3_TRACE_REST_API_PATH_PREFIX = f"{_V3_REST_API_PATH_PREFIX}/mlflow/traces"
 _ARMERIA_OK = "200 OK"
+_DATABRICKS_SDK_RETRY_AFTER_SECS_DEPRECATION_WARNING = (
+    "The 'retry_after_secs' parameter of DatabricksError is deprecated"
```

Comment:
can we just use `retry_after_secs`?

Reply:
I don't think that's safe cuz if some other warnings (not deprecation warning) include the same part they will also be filtered